TestIntReg.java

```diff
@@ -0,0 +1,129 @@
+import java.util.Arrays;
```

Comment:
Copyright header missing.

Reply:
WIP, cosmetic aspects will be addressed before transitioning from draft PR for ready-for-review.